File: JUNGOL/Beginner/#1880 암호풀기/암호풀기.py
def main():
    password = input()
    s = input()

    for si in s:
        if si == ' ': print(' ', end='')
        elif si.isupper():
            tmp = ord(si)-ord('A')
            print(password[tmp], end='')
        else:
            tmp = ord(si)-ord('a')
            print(password[tmp], end='')

main()

File: JUNGOL/Beginner/#1880 암호풀기/암호풀기.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	string password; getline(cin, password);
	string s; getline(cin, s);

	for(int i=0; i<s.length(); i++){
		if(s[i] == ' ') printf(" ");
		else if(isupper(s[i])){
			int tmp = s[i]-'A';
			printf("%c", toupper(password[tmp]));
		}
		else{
			int tmp = s[i]-'a';
			printf("%c", password[tmp]);
		}
	}
}
